pkg/controlplane/control/manager.go

```diff
@@ -164,6 +264,23 @@ func serviceChanged(svc1, svc2 *v1.Service) bool {
 	return false
 }
 
+func conditionChanged(conditions *[]metav1.Condition, cond *metav1.Condition) bool {
```

Comment:
Maybe this should be in  more common package

Reply:
But it used only by controlplane/control